src/r_util.cpp

```diff
@@ -504,6 +506,37 @@ namespace rhost {
             return R_FalseValue;
         }
 
+        extern "C" SEXP save_project(SEXP id, SEXP project_name, SEXP dest_dir, SEXP temp_dir) {
```

Comment:
I'm just thinking if we should call it `save_to_project_folder` or something like this

---
In reply to: [74652029](https://github.com/Microsoft/R-Host/pull/78#discussion_r74652029) [](ancestors = 74652029,74644583)